docs/pages/_app.js

```diff
@@ -213,6 +213,7 @@ function AppWrapper(props) {
       metadata: '',
       name: 'MUI X',
       versions: [
+        { text: 'v7.0.0-alpha.0', href: `https://next.mui.com${languagePrefix}/x/introduction/` },
```

Comment:
WDYT about going with `v7-next` or `v7-prerelease` as the text? 🤔 
Updating these versions with every release would be a nightmare. 🙈

Reply:
I don't think that the version bump on release is that big of a deal, but I am good with both approaches. It's a bit we don'y need to take care then. Agree!